app/models/portfolio_item.rb

```diff
@@ -5,7 +5,7 @@ class PortfolioItem < ApplicationRecord
 
   default_scope -> { kept }
 
-  has_many :icons, :dependent => :destroy
+  has_many :icons, :as => :iconable, :inverse_of => :iconable, :dependent => :destroy
```

Comment:
Do we need `:inverse_of` if there is no other non polymorphic association of `portfolio_item` on the icon model? ( Same question on `Portfolio`. )

Reply:
Spoke in PM about it - it's related to [rubocop](https://rubydoc.info/gems/rubocop/0.69.0/RuboCop/Cop/Rails/InverseOf).